quick-start-with-tidb.md

```diff
@@ -57,10 +57,10 @@ As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB in
         {{< copyable "shell-regular" >}}
 
         ```shell
-        tiup playground v4.0.0 --db 2 --pd 3 --kv 3 --monitor
+        tiup playground {version} --db 2 --pd 3 --kv 3 --monitor
         ```
 
-        The command downloads a v4.0.0 cluster to the local machine and starts it. `--monitor` means that the monitoring component is also deployed.
+        The command downloads a {version} cluster to the local machine and starts it，such as v4.0.11. `--monitor` means that the monitoring component is also deployed.
```

Comment:
I think it is better if it shows v4.0.11 in the example and then says you can change it. This makes the command copy+pasteable.

It is ideally updated as new versions are released, but being stuck on 4.0.11 for a little while is at least better than 4.0.0.